profile/README.md

```diff
@@ -59,7 +59,7 @@ Interested in joining the team? See our [open roles](https://www.warp.dev/career
 
 ## Support and Questions
 
-1. See our [docs](https://docs.warp.dev) for a walkthrough of the features within our app.
+1. See our [docs](https://docs.warp.dev/help/known-issues) for a walkthrough of the features within our app.
```

Comment:
known issues isn't really 'features' within the app, so i think we should reword this or keep original link to just the main docs page.

Reply:
Gonna do this for warp repo too